packages/dm-core-plugins/src/attribute-selector/AttributeSelectorPlugin.tsx

```diff
@@ -19,89 +19,89 @@ import {
 export const AttributeSelectorPlugin = (
   props: IUIPlugin & { config?: TAttributeSelectorConfig }
 ): JSX.Element => {
-  const { idReference, config, onSubmit } = props
+  const { idReference, config } = props
   const internalConfig: TAttributeSelectorConfig = {
     childTabsOnRender: true,
     asSidebar: false,
     items: [],
     ...config,
   }
   const [entity, isLoading, _, error] = useDocument<TGenericObject>(idReference)
-  const [selectedView, setSelectedView] = useState<number>(0)
+  const [selectedView, setSelectedView] = useState<string>()
```

Comment:
should this be initialized to empty string? Otherwise I think type is `string | undefined`

Reply:
That is implicit typing for the "useState" with no initial value. But added explicitly now as well